src/App.js

```diff
@@ -1,13 +1,20 @@
 import "./App.css";
-import { LandingPage } from "./pages";
+import {ProductListing} from "./pages"
+import {Cart} from "./pages"
 import {Routes , Route} from "react-router-dom"
+import { Wishlist } from "./pages/wishlist/Wishlist";
+import { LandingPage } from "./pages";
```

Comment:
All the pages can be imported in single line if you are using barrel export/import